Examples/Examples.xcodeproj/project.pbxproj

```diff
@@ -47,6 +48,7 @@
 
 /* Begin PBXFileReference section */
 		1C40F3312B46118800C00ED5 /* WorldScaleExampleView.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = WorldScaleExampleView.swift; sourceTree = "<group>"; };
+		1CBE20062BED64E10006B55D /* OfflineMapAreasExampleView.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; name = OfflineMapAreasExampleView.swift; path = Examples/OfflineMapAreasExampleView.swift; sourceTree = "<group>"; };
```

Comment:
The path doesn't look right. It appears at the top of the file navigator for me 🤔 can you remove then readd?

<img width="328" alt="image" src="https://github.com/Esri/arcgis-maps-sdk-swift-toolkit/assets/9660181/b6812685-c9bf-4979-b713-5b738d897fe0">

Reply:
Fixed.